### ISSUE TITLE
Changing Batch Size in VGG16 Transfer Learning

### ISSUE THREAD

**Issue by himanshu2612** [renebidart/breakHis#4]
Is there any way to change Batch-size and other hyper-parameters during training using Transfer Learning?

**Comment by renebidart** [comment]
I'm not sure what you mean. If you want to change the batch size or any other Hyperparameters then change it. For example if you're using ft_pre_model() function, then you should go inside it and change whatever you want.

**Comment by himanshu2612** [comment]
`num_train 1321
num_valid 760
steps_per_epoch 82.0
validation_steps 47.0
Found 0 images belonging to 0 classes.
Found 0 images belonging to 0 classes.
Training the top layers only
Epoch 1/5`

I am getting this output when training with ft_pre_model() function. Why It is showing "Found 0 images belonging to 0 classes". I am also not getting the code to seperate classes or to find the label of images.

**Comment by renebidart** [comment]
Something is wrong with the data generator. Check train_loc to see if it contains the images in correct format.

**Comment by yash-623** [comment]
Even i am getting the same output as himanshu2612. Can you share the correct order of locations of files that were generated in your computer?
